File: cloud-storage-spring-java/src/main/java/com/github/sevtech/cloud/storage/spring/bean/DeleteFileResponse.java
package com.github.sevtech.cloud.storage.spring.bean;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class DeleteFileResponse {
	private boolean result;
	private int status;
	private String cause;
	private Exception exception;
}

File: cloud-storage-spring-kotlin/src/main/kotlin/com/github/sevtech/cloud/storage/spring/bean/DeleteFileResponse.kt
package com.github.sevtech.cloud.storage.spring.bean

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
data class DeleteFileResponse(var result: Boolean = false,
                              var status: Int = 0,
                              var cause: String? = null,
                              var exception: Exception? = null
) : BaseResponse(status, cause, exception)